src/test/java/com/limechain/trie/structure/TrieStructureTest.java

```diff
@@ -0,0 +1,265 @@
moCaFbXm6y/v9/7d3JtCyFOUdL1B8LCIKBFCIKOuJC0Q85CiIISRIDAEOm0EOCEJ8AQIEJWyGRWSTnSDLY5flAMEHCBgRHwiPJTzWIA+QRfZV2Rdl1Un/mnxN3b7dc3umv+478+Zf59zbM93V1V/9uqf7q399VY0Th6OJw0sEtqWNN954TAQJeXDsKCc/bJJ9io5BfuaQjB1lojjMQYyXOMmxg563N85rn7EdZxlBnWGt999/v5lfuLz33nvH1ImoHpvGhwh3Kxen12wmEofRALYNu0zsZvjqBhtskG3DFjoKiLIhQh/72I/jxHPM2+gAHGTr7Cg0OFpJGWaDLWHGfKBMLWTr4iUNGEtF56fbNRAPd43LpGOBqY7iDod4O40lJREQARFok4AE+TZp61giIAKDTgDf1sRnfDSE9+nTp6e+KN8Z/Uie/JQy+LFsZ18LeMH/ZFRtPp1++uljjmH7IRTbiFimurRy2B7/4U+bOI3/Htsb54s/Uw9E8alTp3bOPffcMVPU5O3jO1NlxuUy7Q6jakk2Opbt8ZQ2TPMT70Mgjo3k7acdQbCKlZePZE8NKfj3zDPPZPvk60/QkY20jbfxGSHeUi/XAG2ebn49kfFF7TLaOQRVKYmACIhAWwSqCvJzYVByY0zT/PPPHxLRJSQ3u7DAAgvYai1FQAREoDUCifgeEoc3JM5vSJzj7F6UzGUell566XF2JFHV4UMf+lBgP/JwD/vwhz8cPvaxj4W55557XH5bkTj34e677w6LLLJIWHHFFQP3v0FKSQROSIadhoUWWii1MbYtiX4J88wzT8Ym3pY4wCm75ZZbLl6dfk4c55AI9+ETn/jEmG1JQyQkczyGP/uzPxuzHhuSxk1YeeWVu7Ics5PDl16vAYdDqggREAERaJzAF7/4xTBr1qyQTCEQVltttcaP1+0A2IE9ibgRko7Qblm1TQREQAQaI5CIFiERkFM/HN/1/e9/f8AvTaaWCR/96EfHHRd/Fd+Y/fCT55prrpAIruN85XhH5I5EwA7JO47StsSyyy6bHifOM9mfqbPZN998840xB/990UUXHWcz/nIyIiAkQnpYYoklxuzTTzvCmCYdHmPKavpLr9dA0/aofBEQARGoSwBNCn1pypQpqf5SVp4E+TIyWi8CIiACIiACIiACIiACTgQkyDuBVDEiIAIiIAIiIAIiIAIiMKAEJMgP6ImRWSIgAiIgAiIgAiIgAqNHQIL86J1z1VgEREAEREAEREAERGC0CEiQH63zrdqKgAiIgAiIgAiIgAgMMAEJ8gN8cmSaCIiACIiACIiACIiACDgQkCDvAFFFiIAIiIAIiIAIiIAIiIAHAQnyHhRVhgiIgAiIgAiIgAiIgAgMLgEJ8oN7bmSZCIiACIiACIiACIjAiBGQID9iJ1zVFQEREAEREAEREAERGDkCEuRH7pSrwiIgAiIgAiIgAiIgAoNKQIL8oJ4Z2SUCIiACIiACIiACIiACPgQkyPtwVCkiIAIiIAIiIAIiIAIiUJuABPnaCFWACIiACIiACIiACIiACAw0AQnyA316ZJwIiIAIiIAIiIAIiMAoEZAgP0pnW3UVAREQAREQAREQAREYRQIS5EfxrKvOIiACIiACIiACIiACA0lAgvxAnhYZJQIiIAIiIAIiIAIiIAJuBCTIu6FUQSIgAiIgAiIgAiIgAiJQj4AE+Xr8tLcIiIAIiIAIiIAIiIAIDDoBCfKDfoZknwiIgAiIgAiIgAiIwMgQkCA/MqdaFRUBERABERABERABERhRAhLkR/TEq9oiIAIiIAIiIAIiIAKDR0CC/OCdE1kkAiIgAiIgAiIgAiIgAp4EJMh70lRZIiACIiACIiACIiACIlCDgAT5GvC0qwiIgAiIgAiIgAiIgAgMAQEJ8kNwkmSiCIiACIiACIiACIjAaBCQID8a51m1FAEREAEREAEREAERGF0CtQT5Cy+8MEyZMmV06anmIiACIiACIiACIiACIuBI4N///d/DvffeG2644Yaw2mqrOZbce1GzZs0KdBAsv/zy4eijj+69AO0hAiIgAiIgAiIgAiIgAiIwjsBbb70VNtpoo1RXf+ONN8ZttxVzdZJkX+aff/6Akq8kAiIgAiIgAiIgAiIgAiLgT2CQBHn/2qlEERABERABERABERABERABAt17FuTXXnvt8L73vU/0REAEREAEREAEREAEREAEHAgQlf7SSy8NVIT8QgstlEbKO1RPRYiACIiACIiACIiACIjAyBP44x//GGbMmNFfhPxrr70WFlhggZGHKAAiIAIiIAIiIAIiIAIi4EFAc8h7UFQZIiACIiACIiACIiACIjC4BGrNIS9BfnBPrCwTAREQAREQAREQAREYPgIS5IfvnMliERABERABERABERABEeiFgAT5XmgprwiIgAiIgAiIgAiIgAg0SECCfINwVbQIiIAIiIAIiIAIiIAIDAABCfIDcBJkggiIgAiIgAiIgAiIgAhAQIK8rgMREAEREAEREAEREAERmLMJSJCfs8+vaicCIiACIiACIiACIjBEBCTID9HJkqkiIAIiIAIiIAIiIAIi0AcBCfJ9QNMuIiACIiACIiACIiACItAEAQnyTVBVmSIgAiIgAiIgAiIgAiIwOAQkyA/OuZAlIiACIiACIiACIiACI05AgvyIXwCqvgiIgAiIgAiIgAiIwBxPQIL8HH+KVUEREAEREAEREAEREIFhISBBfljOlOwUAREQAREQAREQAREQgf4ISJDvj5v2EgEREAEREAEREAEREAF3AhLk3ZGqQBEQAREQAREQAREQAREYKAIS5AfqdMgYERABERABERABERCBUSYgQX6Uz77qLgIiIAIiIAIiIAIiMAoEJMiPwllWHUVABERABERABERABIaCgAT5oThNMlIEREAEREAEREAEREAE+iYgQb5vdNpRBERABERABERABERABHwJSJD35anSREAEREAEREAEREAERGDQCEiQH7Q
```

Comment:
why? 😄

Reply:
the image is present in 2 places

Reply:
> why? 😄

I forgot to remove this note, it was just a reaction. 😄  I'm kind of very keen on the visual capabilities of JavaDoc, and I think that's a sensible use case.

Reply:
> the image is present in 2 places

If you mean the other test above, it's a different picture. If you don't mean this, I kind of don't get the comment. 

But yeah, the idea being that images visualise what the expected trie at the end looks like, as an additional clarification to the assertions.